### DISCUSSION TITLE
openwrt 安装完在左侧找不到图形界面。

### DISCUSSION THREAD

**Discussion by xdtxAM** [messense/aliyundrive-webdav#244] [Q&A] [answered]
### 问题描述

openwrt 安装完在左侧找不到图形界面。 插件安装完在服务列表不显示出来。
我使用命令行是可以运行的，使用命令aliyundrive-webdav --auto-index --port 8111 --refresh-token xxxxxxx  打开 web 界面之后，如果关闭掉浏览器的命令航窗口，这个 web 界面也就掉了

### 重现步骤

1、使用命令aliyundrive-webdav --auto-index --port 8111 --refresh-token xxxxxxx 
2、进入local:8111
3、关闭命令行窗口，web 会连接失败

### 版本

1.1.0

### 运行平台

树莓派 aarch64_cortex-a72 

### 日志

无报错

**Comment by messense** [comment, marked as answer]
https://forum.openwrt.org/t/how-do-i-completely-clear-any-luci-caches-and-restart-it/74241
